web-citizen-reporting/web-citizen-reporting-template/src/components/ReportAnswersStep.tsx

```diff
@@ -87,17 +86,14 @@ function QuestionDescription({
 
 function ReportAnswersStep() {
   const { formId } = Route.useParams();
-  const { data: citizenReportFoms } = useSuspenseQuery(formsOptions());
   const [loading, setLoading] = React.useState(false);
   const { onUpload, progresses, uploadedFiles, isUploading } = useUploadFile({
     defaultUploadedFiles: [],
   });
 
-  const citizenReportForm = citizenReportFoms.find((f) => f.id === formId);
-
-  if (citizenReportForm === undefined) {
-    throw notFound({ throw: false });
-  }
+  const citizenReportForm = useLoaderData({
```

Comment:
please use `Route.useLoaderData()` this will return `FormModel | undefined` so we would need this :
```js
if (citizenReportForm === undefined) {
    throw notFound({ throw: false });
}
```